File: python/string_060919..py
#sourcecode
n,c=input(),0
l=list(map(str,input().split()))
for i in range(len(n)-1):
    if len(l[i])>len(l[i+1]):
        c=1
if c==0:
    print("Yes")
else:
    print("No")
#check if the input len is in asc. ord
#if yes print yes and if else print no

'''
Sample IP:
5
a ab abc

Sample OP:
Yes
'''

File: this_is_in_c/string_060919.c
//sourcecode
#include<stdio.h>
#include<stdlib.h>
main()
{
char x,y;
int p=1;
scanf("%c %c",&x,&y);
if(x>y)
p=-1;
for(char a=x;;a+=p)
{if(a!='a'&&a!='e'&&a!='i'&&a!='o'&&a!='u')
printf("%c ",a);
if(a==y)
break;}
}
//print consonants in range x and y

/*
Sample IP:
e a

Sample OP:
d c b
*/
